api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/health/ApiCatalogHealthIndicator.java

```diff
@@ -22,19 +24,40 @@
  */
 @Component
 @RequiredArgsConstructor
+@Slf4j
 public class ApiCatalogHealthIndicator extends AbstractHealthIndicator {
+    private static String AUTHENTICATION_SERVICE_ID = "apiml.authorizationService.zosmfServiceId";
+    private static String AUTHENTICATION_SERVICE_PROVIDER = "apiml.authorizationService.provider";
+    private static String ZOSMF = "zosmf";
 
     private final DiscoveryClient discoveryClient;
 
     @Override
     protected void doHealthCheck(Health.Builder builder) {
         String gatewayServiceId = CoreService.GATEWAY.getServiceId();
+        String authServiceId = ZOSMF;
+        String authServiceProvider = ZOSMF;
```

Comment:
Would it be possible to move the check for whether authorization is up at least to separate private function?